spec/classes/init_spec.rb

```diff
@@ -541,6 +563,7 @@
         'host_aliases' => ['myhost2','loghost'],
       },
     } } }
+    let(:facts) { { :ipaddress => '10.1.2.3' } }
```

Comment:
The same reason I told before. Some tests started failing if fqdn_ip_address or ipaddress are not set.